### PR TITLE
feat: added home page to video library

### PR DESCRIPTION
**Pull request by kedark152**
### feat: added home page to video library app.

**Preview Link: https://deploy-preview-1--agroplay.netlify.app/**

**Features in this PR**
1. Created common re-usable components such as Navbar, Sidebar.
2. added chips component in Home Page.
3. added video listing using css grids in Home Page.
4. fetching all videos from mock backend using useEffect and Async Await.
5. added blank pages for Explore, History, Liked Videos, Watch Later etc.

image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
+import "./styles/utils/variables.css";
+import { PageRoutes } from "./routes/PageRoutes";
 function App() {
   return (
     <>
-      <div className="App">AgroPlay</div>
+      <div className="App">
+        <PageRoutes />
+      </div>
     </>
```
Comment (fahd23) [inline src/App.js]: Use can this fragment <>, as your are returning only one div

File: src/components/ChipsContainer.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  return (
+    <>
+      <div className="chips fw-bold pd-y-xsm">
+        <div id="basic-chip">All</div>
+        <div id="basic-chip">Tomato farming</div>
+        <div id="basic-chip">Pesticides</div>
+        <div id="basic-chip">Organic Fertilizers</div>
+        <div id="basic-chip">Indian Farmer</div>
+      </div>
+    </>
```
Comment (fahd23) [inline src/components/ChipsContainer.js]: Here as well

**Review by fahd23 — APPROVED** [review]
Code looks good and UI Looks nice

**Review by Kavita539 — APPROVED** [review]
- Code looks good, nicely maintained and also clean
- UI also looks clean and minimal
- Folders are structured well.

File: src/components/ChipsContainer.js (hunk 2 above)
Comment (kedark152) [reply src/components/ChipsContainer.js]: made changes in new commit, Thanks

File: src/App.js (hunk 1 above)
Comment (kedark152) [reply src/App.js]: made changes in new commit, Thanks

**Comment by kedark152** [comment]
Thanks Fahd and Kavita for PR Review.

**Pull request merged**